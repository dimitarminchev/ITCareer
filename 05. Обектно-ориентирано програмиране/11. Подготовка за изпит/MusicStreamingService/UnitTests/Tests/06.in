AddUser Alice 20
AddPlaylist Alice MyFavorites
AddSongToPlaylist Alice MyFavorites A 200 ArtistName Rock Single 01/01/2021
End