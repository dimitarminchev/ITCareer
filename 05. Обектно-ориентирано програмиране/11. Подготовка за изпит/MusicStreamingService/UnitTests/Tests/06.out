Created User Alice!
Created Playlist MyFavorites for User Alice!
Title should be between 2 and 100 characters!